hopsworks-common/src/main/java/io/hops/hopsworks/common/project/ProjectController.java

```diff
@@ -1275,6 +1296,36 @@ public void removeProject(String userMail, int projectId, String sessionId) thro
           cleanupLogger.logError(ex.getMessage());
         }
 
+        
+        List<ProjectTeam> reconstructedProjectTeam = new ArrayList<>();
+        try {
+          for (HdfsUsers hdfsUser : hdfsUsersController.getAllProjectHdfsUsers(projectName)) {
+            Users foundUser = userFacade.findByUsername(hdfsUser.getUsername());
+            if (foundUser != null) {
+              reconstructedProjectTeam.add(new ProjectTeam(toDeleteProject, foundUser));
+            }
+          }
+        } catch (Exception ex) {
+          // NOOP
```

Comment:
@kouzant when is this exception thrown?

Reply:
This method is a *forced* project deletion. It should delete as much as possible without failing. In that line, an exception might be thrown from the facades.